src/uct/ib/mlx5/dc/dc_mlx5.c

```diff
@@ -140,6 +140,11 @@ ucs_config_field_t uct_dc_mlx5_iface_config_sub_table[] = {
      ucs_offsetof(uct_dc_mlx5_iface_config_t, dcis_initial_capacity),
      UCS_CONFIG_TYPE_ULUNITS},
 
+    {"FULL_HANDSHAKE_LATENCY", "140ns",
+     "DC Full Handshake extra latency",
+     ucs_offsetof(uct_dc_mlx5_iface_config_t, fhs_latency),
+     UCS_CONFIG_TYPE_TIME_UNITS},
```

Comment:
Is there dependency on `iface_attr->latency` ? then maybe we could calculate default value based on it.

Reply:
instead of having special configuration for FULL_HANDSHAKE_LATENCY, can we say that latency is increased 2x (as calculated in uct_ib_iface_query)?